ipatests/test_integration/test_sudo.py

```diff
@@ -713,3 +714,43 @@ def test_category_all_validation_runasgroup(self):
                                           '--groups', 'testgroup2'],
                                           raiseonerr=False)
         assert result.returncode != 0
+
+    @pytest.mark.xfail(
+        osinfo.id == 'fedora' and osinfo.version_number <= (30,),
```

Comment:
Hi @wladich 
According to [bodhi](https://bodhi.fedoraproject.org/updates/FEDORA-2019-1575ccd38c), the fix is available in sssd 2.2.0 in fedora 30 since June 25. Shouldn't the check be `osinfo.version_number < (30,)` rather than `<=` ?

Reply:
good point, thank you